packages/design-system/src/Box/Box.stories.tsx

```diff
@@ -0,0 +1,836 @@
+      },
+      description: "Border right width of the box",
+    },
+    borderRightStyle: {
+      control: {
+        type: "text",
+      },
+      description: "Border right style of the box",
+    },
+    borderRightColor: {
+      control: {
+        type: "color",
+      },
+      description: "Border right color of the box",
+    },
+    borderBottom: {
+      control: {
+        type: "text",
+      },
+      description: "Border bottom of the box",
+    },
+    borderBottomWidth: {
+      control: {
+        type: "text",
+      },
+      description: "Border bottom width of the box",
+    },
+    borderBottomStyle: {
+      control: {
+        type: "text",
+      },
+      description: "Border bottom style of the box",
+    },
+    borderBottomColor: {
+      control: {
+        type: "color",
+      },
+      description: "Border bottom color of the box",
+    },
+    borderBottomLeftRadius: {
+      control: {
+        type: "text",
+      },
+      description: "Border bottom left radius of the box",
+    },
+    borderBottomRightRadius: {
+      control: {
+        type: "text",
+      },
+      description: "Border bottom right radius of the box",
+    },
+    borderLeft: {
+      control: {
+        type: "text",
+      },
+      description: "Border left of the box",
+    },
+    borderLeftWidth: {
+      control: {
+        type: "text",
+      },
+      description: "Border left width of the box",
+    },
+    borderLeftStyle: {
+      control: {
+        type: "text",
+      },
+      description: "Border left style of the box",
+    },
+    borderLeftColor: {
+      control: {
+        type: "color",
+      },
+      description: "Border left color of the box",
+    },
+    borderX: {
+      control: {
+        type: "text",
+      },
+      description: "Border left and right of the box",
+    },
+    borderY: {
+      control: {
+        type: "text",
+      },
+      description: "Border top and bottom of the box",
+    },
+    position: {
+      control: {
+        type: "text",
+      },
+      description: "Position of the box",
+    },
+    zIndex: {
+      control: {
+        type: "text",
+      },
+      description: "Z-index of the box",
+    },
+    top: {
+      control: {
+        type: "text",
+      },
+      description: "Top of the box",
+    },
+    right: {
+      control: {
+        type: "text",
+      },
+      description: "Right of the box",
+    },
+    bottom: {
+      control: {
+        type: "text",
+      },
+      description: "Bottom of the box",
+    },
+    left: {
+      control: {
+        type: "text",
+      },
+      description: "Left of the box",
+    },
+    boxShadow: {
+      control: {
+        type: "text",
+      },
+      description: "Box shadow of the box",
+    },
+  },
+} as Meta<typeof Box>;
+
+type Story = StoryObj<typeof Box>;
+
+export const BoxStory: Story = {
+  name: "Box",
+  args: {
+    bg: "var(--ads-v2-color-bg-brand)",
+    width: "100px",
```

Comment:
Is it possible to support ADS tokens? How can we make sure that when using this component, tokens(color, spacing, sizing and not custom values will be used?

Reply:
That's a really great suggestion. Let me see how easy to make it happen.

Reply:
I will take this up in a different PR since it requires some more decisions to be made regarding the tokens. Created a task and assigned it to myself.
https://github.com/appsmithorg/appsmith/issues/27995